src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = FigureSupplier.getRandomFigure();
+        }
+
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            figures[i] = FigureSupplier.getDefaultFigure();
+        }
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. In this case, you are not creating a new instance but calling static methods, which is also not recommended. Create an instance of `FigureSupplier` before the loop and use it.